.github/workflows/build-test.yml

```diff
@@ -8,39 +8,39 @@ jobs:
     strategy:
       max-parallel: 3
       matrix:
-        os: [ubuntu-18.04, windows-2019, macos-10.15]
+        os: [ubuntu-24.04, windows-2022, macos-10.15]
         include:
-          - os: ubuntu-18.04
+          - os: ubuntu-24.04
             c-compiler: "gcc"
             cxx-compiler: "g++"
-            itk-git-tag: "d6acfd26bfcdec606d605beb1301bddfb17c05a6"
+            itk-git-tag: "v6.0a02"
             cmake-build-type: "MinSizeRel"
-          - os: windows-2019
+          - os: windows-2022
             c-compiler: "cl.exe"
             cxx-compiler: "cl.exe"
-            itk-git-tag: "d6acfd26bfcdec606d605beb1301bddfb17c05a6"
+            itk-git-tag: "v6.0a02"
             cmake-build-type: "Release"
           - os: macos-10.15
             c-compiler: "clang"
             cxx-compiler: "clang++"
-            itk-git-tag: "d6acfd26bfcdec606d605beb1301bddfb17c05a6"
+            itk-git-tag: "v6.0a02"
             cmake-build-type: "MinSizeRel"
 
     steps:
-      - uses: actions/checkout@v1
+      - uses: actions/checkout@v4
 
-      - name: Set up Python 3.7
-        uses: actions/setup-python@v1
+      - name: Set up Python 3.12
+        uses: actions/setup-python@v5
         with:
-          python-version: 3.7
+          python-version: 3.12
 
       - name: Install build dependencies
         run: |
           python -m pip install --upgrade pip
           python -m pip install ninja
 
       - name: Get specific version of CMake, Ninja
-        uses: lukka/get-cmake@v3.18.3
+        uses: lukka/get-cmake@latest
```

Comment:
We usually chose a specific version. I guess you have chosen this to minimize how frequently we need to update it.

Reply:
Yes.  I'm good with a specific version, what would you suggest?

I waffled on this.  I was trying to minimize the number of times 80+ files need to be modified across 80 different remote modules.

I'll take your advice.

Reply:
I would suggest the latest real tag, which is currently `v3.31.5`.